registerUser.js

```diff
@@ -0,0 +1,70 @@
+let AWS = require("aws-sdk");
+
+AWS.config.update({
+  accessKeyId: process.env.AWS_ACC_KEY_ID,
+  secretAccessKey: process.env.AWS_SECRET
+});
+const COGNITO_CLIENT = new AWS.CognitoIdentityServiceProvider({
+  apiVersion: "2016-04-19",
+  region: "us-east-1"
+});
+const setPasswordPromise = userParams => {
+  return new Promise((resolve, reject) => {
+    COGNITO_CLIENT.adminSetUserPassword(userParams, (error, data) => {
```

Comment:
Can we use [util.promisify](https://nodejs.org/dist/latest-v8.x/docs/api/util.html#util_util_promisify_original) instead?

Reply:
Have tried to use it but wasn't successful but will try to re write again with util.promisify